### PR TITLE
chore: add GitHub <ci> action

### PR DESCRIPTION
**Pull request by nikku**
Preparation to replace Travis CI.

### CODE REVIEW

**Comment by barmac** [comment]
Do we want to run Travis parallel to GitHub Actions in this repo?

**Comment by nikku** [comment]
Yes. We're trialing it right now only.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js ${{matrix.node-version}}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{matrix.node-version}}
+    - name: Cache Node.js modules
+      uses: actions/cache@v2
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
```
Comment (barmac) [inline .github/workflows/ci.yml:82]: Are we going to use Windows-specific path too? Cf. https://docs.npmjs.com/cli/v6/commands/npm-cache#cache

Comment (barmac) [reply .github/workflows/ci.yml:82]: Cf. also https://github.com/actions/cache/blob/main/examples.md#node---npm

Comment (barmac) [reply .github/workflows/ci.yml:82]: [This one](https://github.com/actions/cache/blob/main/examples.md#using-multiple-systems-and-npm-config) could be re-used everywhere.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{matrix.node-version}}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-${{matrix.node-version}}-
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+    - name: Install dependencies
+      run: npm ci
```
Comment (barmac) [inline .github/workflows/ci.yml:30]: As discussed via Slack, this part can be re-used across all the platforms if we adjust the cache directory.

**Comment by barmac** [comment]
I left some comments regarding re-use of the code and cache, but other than that it looks good 👍

**Review by barmac — APPROVED** [review]

**Pull request merged**